src/app/styles/components/tabs.scss

```diff
@@ -0,0 +1,30 @@
+.tab {
```

Comment:
Can you fit all 3 tabs without overflow in mobile layout? You can decrase padding inside tabs, font-size, padding outside list.

Reply:
I have not worked on them on small screens, as Ularbek now needs a vendor page. I will do it in the next pull request

Reply:
I promise